src/main/com/yetanalytics/lrs/pedestal/interceptor/xapi/statements/attachment/response.cljc

```diff
@@ -1,23 +1,21 @@
 (ns com.yetanalytics.lrs.pedestal.interceptor.xapi.statements.attachment.response
   (:require
    [clojure.core.async :as a :include-macros true]
+   [com.yetanalytics.lrs.protocol :as lrsp]
    #?@(:clj [[cheshire.core :as json]]
-       :cljs [[goog.string :as gstring]
+       :cljs [[goog.string :refer [format]]
               [goog.string.format]])))
 
 ;; TODO: Dynamic boundary?
 ;; TODO: make this async, work on a servlet output stream
 
-(def fmt #?(:clj format
-            :cljs gstring/format))
```

Comment:
Hmm I guess this won't cause an error after all. In which case `fmt` can be removed elsewhere (such as in the `interceptors.xapi.statements` namespace).

Reply:
Yeah I'm not sure what the original reasoning was for that, perhaps something in a previous version of cljs, or just a mistake. I didn't want to expand the footprint of this too much, but it only exists in that one other place so I'll go change it.